conmon-rs/common/proto/conmon.capnp

```diff
@@ -4,7 +4,8 @@ interface Conmon {
     ###############################################
     # Version
     struct VersionRequest {
-        verbose @0 :Bool;
+        metadata @0 :Data;
```

Comment:
when adding new fields, @ number should be higher so we have backwards compatibility. the order of the fields don't matter, so this could be
```
metadata @1 :Data;
verbose @0 :Bool;
```

Reply:
Makes sense, I'll change it that way.